frontend/src/features/homes/assign-manager.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <q-dialog
+    class="fit"
+    v-model="show"
+    @hide="$emit('close')"
+    @escape-key="$emit('close')"
+  >
+    <q-card class="full-width">
+      <q-card-section>
+        <div class="text-h4">
+          <q-icon name="fa fa-cubes" class="q-mr-md" />
+          {{ $i18n.t("assignManager-header-text") }}
+        </div>
+      </q-card-section>
+
+      <q-separator />
+      <q-card-section>
+        <q-form @submit="onSubmit" ref="assignManager" class="q-gutter-md">
+          <q-select
+            filled
+            multiple
+            v-model="selectedUsers"
+            map-options
+            use-chips
+            stack-label
+            fill-input
+            option-value="value"
+            option-label="label"
+            :rules="[v => (!!v && v.length > 0) || $i18n.t('fieldRequired')]"
```

Comment:
Use the same `validateMaxLength(maxLength)` function in this rule, for readability.